src/Workspaces/CSharp/Portable/CodeStyle/CSharpCodeStyleOptions.cs

```diff
@@ -88,6 +88,12 @@ internal static class CSharpCodeStyleOptions
                 new EditorConfigStorageLocation("csharp_style_expression_bodied_accessors", ParseEditorConfigCodeStyleOption),
                 new RoamingProfileStorageLocation($"TextEditor.CSharp.Specific.{nameof(PreferExpressionBodiedAccessors)}")});
 
+        public static readonly Option<CodeStyleOption<bool>> PreferBraces = new Option<CodeStyleOption<bool>>(
+            nameof(CodeStyleOptions), nameof(PreferBraces), defaultValue: CodeStyleOptions.TrueWithNoneEnforcement,
+            storageLocations: new OptionStorageLocation[]{
+                new EditorConfigStorageLocation("csharp_prefer_braces", ParseEditorConfigCodeStyleOption),
```

Comment:
Perhaps csharp_prefer_braces_around_single_line_bodies? I wasn't sure if this was to prefer braces instead of expression bodied stuff until reading the code.